crates/rpl_match/src/ty.rs

```diff
@@ -80,7 +80,51 @@ impl<'pcx, 'tcx> MatchTyCtxt<'pcx, 'tcx> {
             //         && self.match_generic_args(args, alias.args)
             // },
             (pat::TyKind::Bool, ty::Bool) => true,
-            _ => false,
```

Comment:
@stuuupidcat @TheVeryDarkness Do you think this is too noisy?

Reply:
Good choice.

Reply:
LGTM